packages/sqs/lib/utils/sqsMessageDeserializer.ts

```diff
@@ -10,9 +10,10 @@ export const deserializeSQSMessage = <T extends object>(
   errorProcessor: ErrorResolver,
 ): Either<MessageInvalidFormatError | MessageValidationError, T> => {
   try {
-    return {
-      result: type.parse(JSON.parse(message.Body)),
-    }
+    const result = JSON.parse(message.Body)
+    type.parse(result)
+
+    return { result }
```

Comment:
I didn't find a better way to do it, I tried to use `zod.passthrough()` but it is only available under `z.object` object but can't be applied to `ZodType` or `ZodSchema`